portability-spi-transfer/src/main/java/org/dataportabilityproject/spi/transfer/TransferCopier.java

```diff
@@ -0,0 +1,18 @@
+package org.dataportabilityproject.spi.transfer;
+
+import java.io.IOException;
+import org.dataportabilityproject.spi.transfer.provider.TransferServiceProvider;
+import org.dataportabilityproject.spi.transfer.provider.TransferServiceProviderRegistry;
+import org.dataportabilityproject.types.transfer.PortableType;
+import org.dataportabilityproject.types.transfer.auth.AuthData;
+import org.dataportabilityproject.types.transfer.models.DataModel;
+
+public interface TransferCopier {
```

Comment:
Add java doc, something to the effect of in memory copier (since we may break copying up later to take part between two different binaries)

Reply:
Done